app/models/ThangType.coffee

```diff
@@ -392,6 +392,12 @@ module.exports = class ThangType extends CocoModel
     props = _.without props, 'canCast', 'spellNames', 'spells'
     for stat, modifiers of itemConfig.stats ? {}
       stats[stat] = @formatStatDisplay stat, modifiers
+      if @heroStatModifier? and stats['maxHealth']? and !isNaN(stats['maxHealth'].display)
+        healthModifier = @heroStatModifier.health.absolute
+        totalHealth = Math.round(healthModifier * stats['maxHealth'].display)
```

Comment:
Style is to do `stats.maxHealth.display` (direct property access with dot notation instead of string index, since we know the property name here).